### PR TITLE
Code completion for `import`s v1

### PR DESCRIPTION
**Pull request by simerplaha**
- Towards #98 
- Imports already contain `SourceIndex` so supporting them first.

VSCode demo

https://github.com/alephium/ralph-lsp/assets/1773953/5b9f295b-7280-462c-a94a-6bcf9fdc8553

### CODE REVIEW

File: presentation-compiler/src/test/scala/org/alephium/ralph/lsp/pc/completion/ImportCompleterSpec.scala [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            Suggestion.Field(
+              label = suggestion,
+              insert = suggestion,
+              detail = "",
+              documentation = ""
+            )
+        }
+
+      actual should contain theSameElementsAs expected
+    }
```
Comment (tdroxler) [inline presentation-compiler/src/test/scala/org/alephium/ralph/lsp/pc/completion/ImportCompleterSpec.scala:104]: how does it react if I have: `TestCodeCompleter("""import "std/fungible@@" """)` ?

Comment (simerplaha) [reply presentation-compiler/src/test/scala/org/alephium/ralph/lsp/pc/completion/ImportCompleterSpec.scala:104]: The editor does the filtering. We just return them all.


https://github.com/alephium/ralph-lsp/assets/1773953/7afd34d6-b097-46fe-8086-41ae600474ba

Comment (tdroxler) [reply presentation-compiler/src/test/scala/org/alephium/ralph/lsp/pc/completion/ImportCompleterSpec.scala:104]: sweet

**Review by tdroxler — APPROVED** [review]
So cool :rocket:

**Pull request merged**